jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpConnection.java

```diff
@@ -907,6 +942,8 @@ private Callback resetCallback()
 
         private void release()
         {
+            if (_callback != null)
```

Comment:
If this check fails, all kind of nasty stuff is going to happen. Since this should never happen, this may be okay?

Reply:
Isn't that what ISE is for?